flip-master-libs-to-stable

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash -e
```

Comment:
Shouldn't this file name start with an `_`? I may have a wrong understanding of why script names start with an `_` in this repo or not, but my understanding is that if it's quite uncommon that one of us would call that particular script directly, it should start with an `_`. In this particular case we would call the `_do-batch...` script instead of this one directly.

Reply:
Maybe?